src/integrationTest/java/uk/gov/hmcts/darts/common/repository/ExternalObjectDirectoryRepositoryTest.java

```diff
@@ -71,13 +69,11 @@ void testGetDirectoryIfMediaDateBeyond24Hours() throws Exception {
         // setup the test data
         generateDataWithMediaForInbound(setupHoursBeforeCurrentTime);
 
-        int hourDurationBeyondHours = setupHoursBeforeCurrentTime; // which no records are
-
-        // excerise the logic
+        // exercise the logic
```

Comment:
No need for these comments all over the place. They offer no help to a developer.